machineconfiguration/v1/types.go

```diff
@@ -844,7 +844,8 @@ type ContainerRuntimeConfiguration struct {
 	// +optional
 	OverlaySize *resource.Quantity `json:"overlaySize,omitempty"`
 
-	// defaultRuntime is the name of the OCI runtime to be used as the default.
+	// defaultRuntime is the name of the OCI runtime to be used as the default for containers.
+	// Options are runc and crun. Defaults to crun.
```

Comment:
This field description should probably be updated to be a bit more consistent with the current way we document fields like this in OpenShift.

Something along the lines of:
```
defaultRuntime is an optional field used to configure the OCI runtime used by default for containers.
Allowed values are `runc`, `crun`, and omitted (`""`).
When set to `runc`, ...
When set to `crun`, ...
When omitted, this means no opinion and the platform is left to choose a reasonable default, which is subject to change over time. Currently, the default is `crun`.
```